vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridExpandAllIT.java

```diff
@@ -46,6 +50,20 @@ public void addNewItemAfterCollapseAndExpand() throws InterruptedException {
         assertNewChild();
     }
 
+    @Test
+    public void recalculateWidthsAfterExpend() throws InterruptedException {
+        open();
+        TreeGridElement grid = $(TreeGridElement.class).get(1);
+        WebElement expandToggleElement = grid.getExpandToggleElement(1, 0);
+        int widthBeforeExpend = grid.findElements(By.tagName("vaadin-grid-cell-content")).get(2).getSize().getWidth();
```

Comment:
"2" here is a magic number. Rather use something else, like `vaadin-grid-tree-toggle` element if it works for this case. If it doesn't, you should add a dedicated element for this measuring purpose.

Reply:
thank you for your correct. changed it.